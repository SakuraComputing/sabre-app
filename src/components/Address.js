import React from 'react';

const Address = ( { address } ) => (
    <table className='address-panel'>
        <tbody>
            <tr>{address.addressName}</tr>
            <tr>{address.addressLine1}</tr>
            <tr>{address.addressLine2}</tr>
            <tr>{address.town}</tr>
            <tr>{address.county}</tr>
            <tr>{address.telephone}</tr>
        </tbody>
    </table>
);
export default Address;
