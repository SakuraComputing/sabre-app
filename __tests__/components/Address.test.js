import React from 'react';
import { shallow } from 'enzyme';
import Address from '../../src/components/Address';

const address = {
    addressNo: 1,
    addressName: "Alex Delarge",
    addressLine1: "Broadmoor",
    addressLine2: "",
    town: "Crowthorne",
    county: "Berkshire",
    telephone: "01344 77 3111"
};

describe('Address component',() => {
   it('should render correctly',() => {
      const wrapper = shallow(<Address address={address}/>)
      expect(wrapper).toMatchSnapshot();
   });
});