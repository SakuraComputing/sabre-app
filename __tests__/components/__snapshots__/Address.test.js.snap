// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Address component should render correctly 1`] = `
<table
  className="address-panel"
>
  <tbody>
    <tr>
      Alex Delarge
    </tr>
    <tr>
      Broadmoor
    </tr>
    <tr />
    <tr>
      Crowthorne
    </tr>
    <tr>
      Berkshire
    </tr>
    <tr>
      01344 77 3111
    </tr>
  </tbody>
</table>
`;
